pkg/api/v1/types.go

```diff
@@ -2292,6 +2292,12 @@ type Binding struct {
 	Target ObjectReference `json:"target"`
 }
 
+// Preconditions must be fulfilled before an operation (update, delete, etc.) is carried out.
+type Preconditions struct {
+	// Specifies the target UID.
+	UID *types.UID `json:"uid,omitempty"`
```

Comment:
OK - thanks (I agree that RV is not good for this purpose - it's good for other purposes, e.g. change my object assuming that a different object didn't change since my computation (=has a given RV)). This was widely used pattern in Omega.